frontend/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@coreui/react": "^4.11.1",
```

Comment:
Can remove this dependency if it is not being used anymore

Reply:
removed!